frontend/src/admin/components/sidebar/sidebar.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import UserList from "./user-list/user-list";
+import RestartServerButton from "./restart/restart-server-button";
+import "./sidebar.scss";
+
+export default function Sidebar() {
```

Comment:
A sidebar is normally a bar that you toggle or make slide in and out of view.
What I have seen of the UI so far, there is no sidebar functionality. Is that something that has been added? If not, then I don't think Sidebar is the correct term to use here